zipkin-ui/test/component_data/trace.test.js

```diff
@@ -262,10 +262,10 @@ describe('convertSuccessResponse', () => {
 
     const timeMarkers = [
       {index: 0, time: ''},
-      {index: 1, time: '3.4000000000000004μ'},
-      {index: 2, time: '6.800000000000001μ'},
-      {index: 3, time: '10.2μ'},
-      {index: 4, time: '13.600000000000001μ'},
```

Comment:
3rd change is the actual UI nicies, like this horrible rendering of fractions of a us :D